esmvaltool/cmor/_fixes/OBS/NCEP_NCAR.py

```diff
@@ -0,0 +1,15 @@
+"""Fixes for NCEP-NCAR"""
```

Comment:
There is no dataset called `NCEP-NCAR`, as far as I know, but just `NCEP` (see [list](https://github.com/ESMValGroup/ESMValTool/issues/232#issuecomment-391988805)).

Reply:
I have one named like that in Jasmin. It was the one I used for testing, but probably we should remove it in the main branch

Reply:
Are you sure this is not from `obs4mips`.
Otherwise I would remove it to avoid confusion.